docs/docs-content/deployment-modes/agent-mode/manage-agent/localui-theming.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: "Customize Local UI Theme"
+title: "Customize Local UI Theme - Agent mode"
+description: "Instructions for customizing Local UI theme for hosts deployed in agent mode."
+hide_table_of_contents: false
+sidebar_position: 100
+tags: ["edge"]
+---
+
+Palette offers the option to customize the Local UI web interface. You can change the color of the sidebar as well as
```

Comment:
Maybe I'm missing something but outside of an EdgeForge context. How does a host become a localUI host? It's as simple as the host coming up, registering with Palette and accessing port 5080?

Reply:
Yes. Local UI is enabled by default. Whether you register with Palette or not, as long as you didn't disable it in your user data, it's going to be available at port 5080.